demo/vl-map-wms-layer.html

```diff
@@ -40,35 +40,53 @@
       </vl-map>
     </vl-demo>
 
-    <vl-demo data-vl-title="Kaart met Image WMS kaartlaag met custom SLD">
-      <vl-map id="map-with-image-wms-layer">
+    <vl-demo data-vl-title="Kaart met WMS kaartlaag met custom Styled Layer Descriptor">
+      <vl-map>
         <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
-        <vl-map-image-wms-layer
+        <vl-map-tiled-wms-layer
           data-vl-name="Overstromingsgevaarkaarten"
           data-vl-version="1.1.0"
-          data-vl-opacity="0.6"
-          data-vl-url="https://geoservice.waterinfo.be/Overstromingsgevaarkaarten-PLUVIAAL/wms"
-          data-vl-layers="Overstromingsgevaarkaarten-PLUVIAAL:overstroombaar_gebied_PLU_noCC"
-          data-vl-sld-body='<StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0.0" xsi:schemaLocation="http://www.opengis.net/sld StyledLayerDescriptor.xsd">
-              <NamedLayer>
-                  <Name>Overstromingsgevaarkaarten-PLUVIAAL:overstroombaar_gebied_PLU_noCC</Name>
-                  <UserStyle>
-                      <FeatureTypeStyle>
-                          <Rule>
-                              <RasterSymbolizer>
-                                  <Opacity>1</Opacity>
-                                  <ColorMap type="values">
-                                      <ColorMapEntry color="#800080" quantity="10.0"/>
-                                      <ColorMapEntry color="#FFFFFF" quantity="100.0" opacity="0"/>
-                                      <ColorMapEntry color="#FFFFFF" quantity="1000.0" opacity="0"/>
-                                  </ColorMap>
-                              </RasterSymbolizer>
-                          </Rule>
-                      </FeatureTypeStyle>
-                  </UserStyle>
-              </NamedLayer>
-          </StyledLayerDescriptor>'>
-        </vl-map-image-wms-layer>
+          data-vl-opacity="0.7"
+          data-vl-url="https://geoservice.waterinfo.be/wms"
+          data-vl-layers="Overstromingsgevaarkaarten-PLUVIAAL:overstroombaar_gebied_PLU_noCC,Overstromingsgevaarkaarten-FLUVIAAL:overstroombaar_gebied_FLU_noCC">
+          <vl-map-wms-style
```

Comment:
kunt ge dan nu ook niet zeggen dat de vl-map-wms-style zijnen "innerHTML" leest ipv dat als attribuut door te geven?

Reply:
Had ik geprobeerd en niet direct gelukt, maar zoals Sander al aanhaalde, is het niet zo slim om xml en html te mixen (in de StyledLayerDescriptor xml is bvb meer toegestaan qua casing)

Reply:
HTML is XML

Reply:
of beter, XHTML is XML

Reply:
Maar XML is geen XHTML misschien, dus t zou kunnen dat dat idd niet goed mixed.

Reply:
HTML5 != XHTML en dus niet per se compatibel met XML.

Reply:
Lijkt me inderdaad beter om de XML als child element mee te geven dan als String attribuut.

Reply:
Uw HTML gaat mogelijks invalid worden als je dat zou doen. Misschien werkt het wel als je het wrapped met CDATA, @Pieter-Beckers .